src/inspected-window-helpers/overlay-helpers.js

```diff
@@ -130,9 +130,14 @@ export function setupOverlayHelpers() {
   }
 
   function getColorFromString(string, opacity = 0.4) {
+    const cleanStr = string
+      .split("")
+      .map(l => (/[^0-9a-z]/gi.test(l) ? l.charCodeAt(0) : l)) // replace non-ascii with charCode integer
```

Comment:
Here is a screenshot of the results of the appNames I tested: 

![Screen Shot 2020-04-17 at 10 30 42 AM](https://user-images.githubusercontent.com/4202993/79592003-94e97900-8096-11ea-9d99-a2cccf54869c.png)